samples/ContainerRun/Program.cs

```diff
@@ -35,6 +36,8 @@ static void Main(string[] args)
                         Console.Out.WriteLine("starting container");
                         Console.Out.Flush();
                         container.Start();
+                        try
+                        {
```

Comment:
It's just the viewer... there is a using statement above this line that got cut off.